tests/test_subscriptions.py

```diff
@@ -109,15 +109,17 @@ def test_simple_plan_add_on_creation(self):
 
         self.assertEqual(len(mocurly.backend.plan_add_ons_backend.datastore), 2)
         foo_add_on_backed = mocurly.backend.plan_add_ons_backend.get_object(self.base_backed_plan_data['plan_code'] + '__foo')
-        foo_add_on = filter(lambda add_on: add_on['add_on_code'] == 'foo', self.base_add_on_data)[0]
+        add_ons = filter(lambda add_on: add_on['add_on_code'] == 'foo', self.base_add_on_data)
+        foo_add_on = list(add_ons)[0]
         for k, v in foo_add_on.items():
             if k == 'unit_amount_in_cents':
                 self.assertEqual(foo_add_on_backed[k], dict((curr, str(amt)) for curr, amt in v.currencies.items()))
             else:
                 self.assertEqual(foo_add_on_backed[k], v)
 
         bar_add_on_backed = mocurly.backend.plan_add_ons_backend.get_object(self.base_backed_plan_data['plan_code'] + '__bar')
-        bar_add_on = filter(lambda add_on: add_on['add_on_code'] == 'bar', self.base_add_on_data)[0]
+        add_ons = filter(lambda add_on: add_on['add_on_code'] == 'bar', self.base_add_on_data)
+        bar_add_on = list(add_ons)[0]
```

Comment:
Nit: I feel like, in general, `next(iterator)` serves the purpose better than actually casting things to a list if you're only grabbing the first value.

Reply:
The reason why I did this is because it fails in python2:
```
Traceback (most recent call last):
  File "/Users/yoriy/captricity/mocurly/tests/test_subscriptions.py", line 122, in test_simple_plan_add_on_creation
    bar_add_on = next(add_ons)
TypeError: list object is not an iterator
```
Also because this is a test and not the library code. In the library, I would probably use `six` to make the python2 version return an iterator

Reply:
This doesn't work out of box in python2, because `next(lst)` will raise an exception. I instead made it cleaner by using the versions of dict `keys` and `filter` returned by `six`.